ui/src/app/utils/countries.ts

```diff
@@ -0,0 +1,254 @@
+enum Country {
```

Comment:
Ordered list of countries as we want them to be displayed in the country dropdown